hack/udevd/90-selinux.rules

```diff
@@ -1,5 +1,3 @@
-SUBSYSTEM=="*",SECLABEL{selinux}="system_u:object_r:device_t:s0"
-
```

Comment:
I added this by mistake, when my xattrs implementation failed to report SELinux label on non-labeled device (yet default is always in there). #9788 fixed that